src/topic.ts

```diff
@@ -656,7 +647,7 @@ export class Topic {
   publishJSON(
       json: object, attributesOrCallback?: Attributes|PublishCallback,
       callback?: PublishCallback): Promise<string>|void {
-    if (!is.object(json)) {
+    if (!json || typeof json !== 'object') {
```

Comment:
This kind of check makes me nervous. What are your feelings on using a `try/catch` to wrap the call to `JSON.stringify()` instead?